README.md

```diff
@@ -24,12 +26,12 @@ Clone this repository and build it locally, for this you need `Git`, `Maven` and
 $ git clone https://github.com/porcelli/bc-git-integration-push.git
 $ cd bc-git-integration-push
 $ mvn clean install
-$ mkdir -p $APP_SERVER_HOME/hooks/ && cp target/git-push-2.1-SNAPSHOT.jar $APP_SERVER_HOME/hooks/
+$ mkdir -p $APP_SERVER_HOME/hooks/ && cp target/git-push-2.2-SNAPSHOT.jar $APP_SERVER_HOME/hooks/
 ```
 
 ## How to setup
 
-The application uses a configuration file `.gitremote` that is located under the user home directory, if you don’t want to start from scratch, you can start running the jar file for the first time after the setup above or just as java -jar git-push-2.1-SNAPSHOT.jar, the application will generate a template configuration file in $HOME/.gitremote that you can follow and modify.
+The application uses a configuration file `.gitremote` that is located under the user home directory, if you don’t want to start from scratch, you can start running the jar file for the first time after the setup above or just as java -jar git-push-2.2-SNAPSHOT.jar, the application will generate a template configuration file in $HOME/.gitremote that you can follow and modify.
```

Comment:
rever the version upgrade changes. I'll do it after the merge myself

Reply:
Done